### DISCUSSION TITLE
Purge users - Users who are not accessing remote portal in last N days can be purged.

### DISCUSSION THREAD

**Discussion by naveenk93** [Ylianst/MeshCentral#6536] [General]
Is there any option in MeshCentral to purge the old users, instead of manual removal of user account, can there be any option to keep users who are logged in for last N days/weeks/months.

**Comment by si458** [comment]
Not that I'm aware of no, this would need to be a feature request to add something like this in
OR
Use the websocket as the api and run ur own scripts every so often to check/delete users
You can use meshctrl.js as a basis of your code

**Reply by naveenk93** [reply]
Thanks Simon.
